browsermob-rest/src/main/java/net/lightbody/bmp/proxy/ProxyManager.java

```diff
@@ -224,7 +240,13 @@ private LegacyProxyServer startProxy(LegacyProxyServer proxy, int port) {
         }
 
         try {
-            proxy.start();
+            if (serverBindAddr != null && proxy instanceof BrowserMobProxyServer) {
+                BrowserMobProxyServer bProxy = (BrowserMobProxyServer) proxy;
+                bProxy.start(port, bProxy.getClientBindAddress(), serverBindAddr);
```

Comment:
The clientBindAddress should be null (listen on all interfaces), which is what getClientBindAddress() will return if the proxy isn't started.